pkg/actors/actors.go

```diff
@@ -1648,7 +1648,11 @@ func parseISO8601Duration(from string) (time.Duration, int, error) {
 		}
 		switch name {
 		case "year":
-			duration += time.Hour * 24 * 365 * time.Duration(val)
+			daysPreYear := time.Duration(365)
+			if ((val%4) == 0 && (val%100) != 0) || (val%400) == 0 {
+				daysPreYear = time.Duration(366)
```

Comment:
Please use `time.YearDay()`.

Reply:
The "val" is not a specific year, but a number of years. So, it's not the correct solution....